modules/migration-using-cpma.adoc

```diff
@@ -80,9 +86,8 @@ INFO[29 Aug 19 00:08 UTC] Report:Added: report.json
 INFO[29 Aug 19 00:08 UTC] Report:Added: report.html
 INFO[29 Aug 19 00:08 UTC] Successfully finished transformations
 ----
-<1> `--manifests=false`: Without generating CR manifests
-<2> `Remote host`: Remote mode
-<3> `SSH login`: The SSH user must have `sudo` permissions on the {product-title} 3 cluster in order to access the configuration files.
+<1> The `Remote host` option runs the CPMA in remote mode.
+<2> `SSH login`: The SSH user must have `sudo` permissions on the {product-title} 3 cluster in order to access the configuration files.
```

Comment:
> ! in order to prep
Use “to” in most cases to avoid wordiness.

Perhaps **s/in order to/to** so it's more concise?

Reply:
Ha! Some peer reviewers have forced me to change "to" to "in order to".

Reply:
So unless you or the other reviewers can document the superiority of one usage over the other, it stands. :-)

Reply:
Interesting! :sweat_smile: I imagine the ISG would have the final say, but we can cross that bridge another day since there's some differing opinions. Thanks!